pango/src/attribute.rs

```diff
@@ -14,96 +20,153 @@ use crate::Weight;
 use glib::translate::*;
 
 impl Attribute {
+    #[cfg(any(feature = "v1_44", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v1_44")))]
+    #[doc(alias = "pango_attr_allow_breaks_new")]
+    pub fn new_allow_breaks(allow_breaks: bool) -> Option<Attribute> {
+        unsafe { from_glib_full(ffi::pango_attr_allow_breaks_new(allow_breaks.to_glib())) }
+    }
+
     #[cfg(any(feature = "v1_38", feature = "dox"))]
     #[cfg_attr(feature = "dox", doc(cfg(feature = "v1_38")))]
+    #[doc(alias = "pango_attr_background_alpha_new")]
```

Comment:
@GuillaumeGomez Why didn't your script add those?

Reply:
My script is looking for `sys::` and not `ffi::`. ;)

Another run will be needed then!

Reply:
Oh also! Would be nice to have the C equivalent for the types too.

Reply:
We don't use `sys::` anywhere since before you ran the script :P

Reply:
Hum... Then the bug must be somewhere else.